docker/zotonic-docker.config

```diff
@@ -0,0 +1,11 @@
+%% This file is copied to docker-data/etc-zotonic/config.d/
```

Comment:
As explained above, copying this directly to `/etc/zotonic` is much clearer.

Reply:
Yeah, can do

Reply:
It should be mounted there anyway.

Reply:
As a matter of fact, it's already available in the dev container, so doesn't need to be copied on `docker-compose run` at all.

Reply:
That config file?

My goal with that data folder is that you can zap it and start fresh.
So we have to ensure that the config is copied before we start Zotonic, as it sets the current database connection (and more, if needed in the future).

Reply:
If you don’t mount anything at `/etc/zotonic`, the default config that is built into the container image is used.

Reply:
I would prefer to have (mutable) config outside the container, so that it is easily changed.  (Think of smtp relays etc)

Reply:
I kept the docker-data directory for mapping the zotonic config and security directory.